From 457a8f5fbcbe22535af97ac198f171029d36aebc Mon Sep 17 00:00:00 2001
From: Nolen Johnson <johnsonnolen@gmail.com>
Date: Thu, 17 Feb 2022 13:25:13 -0500
Subject: [PATCH 01/11] Be more explicit about GMS dumpvars

* For most people and in most enviorments these will
  never pop up, but given we now support GMS variants
  and different build types we ought to tell the user
  which one is in use.

Change-Id: I7639f2e741d9b24e3719ef76d7d532c0155cd988
---
 ui/build/dumpvars.go | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ui/build/dumpvars.go b/ui/build/dumpvars.go
index b16c4c8cb..1eb68021a 100644
--- a/ui/build/dumpvars.go
+++ b/ui/build/dumpvars.go
@@ -167,8 +167,8 @@ var BannerVars = []string{
 	"SOONG_SDK_SNAPSHOT_VERSION",
 	"WITH_SU",
 	"WITH_GMS",
-	"PRODUCT_IS_ATV",
-	"PRODUCT_IS_AUTO",
+	"WITH_GMS_CAR",
+	"WITH_GMS_TV",
 	"GMS_MAKEFILE",
 	"MAINLINE_MODULES_MAKEFILE",
 }
-- 
2.37.1

